packages/datastore/__tests__/helpers/fakes/graphqlService.ts

```diff
@@ -730,16 +785,28 @@ export class FakeGraphQLService {
 					}
 				}
 
-				this.notifySubscribers(tableName, type, data, selection, ignoreLatency);
+				this.notifySubscribers(
```

Comment:
This isn't right yet. Isn't the sub only updated when there is data? In the reject case, I don't think an update gets posted. Need to check with the sample app.